force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -14,45 +14,75 @@ import SESSION_END_FIELD from "@salesforce/schema/ServiceSession__c.SessionEnd__
 
 export default class ReviewSessions extends LightningElement {
     @track columns = [];
-    @track data = [];
     @track objectName;
+    _serviceScheduleModel;
 
     @api sessionNameLabel;
     @api sessionStartLabel;
     @api sessionEndLabel;
 
+    @api
+    get serviceScheduleModel() {
+        return this._serviceScheduleModel;
+    }
+    set serviceScheduleModel(value) {
+        // This is a nested object so the inner objects are still read only when using spread alone
+        this._serviceScheduleModel = JSON.parse(JSON.stringify(value));
+        this.labels = Object.assign(
+            {},
+            this.labels,
+            this._serviceScheduleModel.labels.serviceSession
+        );
+        this.getSessions();
```

Comment:
@bethbrains , One thing I noticed is that the datatable does not show all the sessions after the values are entered on Screen1 and the next button is clicked. So to have the data populated in the datatable we will have to click Next and go to screen 3 and then we have to come back to screen 2 to see the new changes. This happens only when the Service Schedule Ends is set to a date.